web/css/css.css

```diff
@@ -0,0 +1,3 @@
+.message-warning {
```

Comment:
Maybe we should call this file `main.css` or even `base.css`. `css.css` is not descriptive